persist/fs/clone/options.go

```diff
@@ -1,3 +1,23 @@
+// Copyright (c) 2018 Uber Technologies, Inc.
```

Comment:
note to future self: https://github.com/m3db/ci-scripts/issues/12 ++

Reply:
Ha, aye, all good.